### PR TITLE
Fix null pointer exception in config.getMapList()

### PR DESCRIPTION
**Pull request by frolic**
Quick bug fix! Someone forgot a null check ;)

### CODE REVIEW

File: src/main/java/org/bukkit/configuration/MemorySection.java [hunk 1]
```diff
@@ -787,6 +787,11 @@ public boolean isList(String path) {
         }
 
         List<Object> list = getList(path);
+
```
Comment (RobinvanderVliet) [inline src/main/java/org/bukkit/configuration/MemorySection.java:790]: You should remove this whitespace.

File: src/main/java/org/bukkit/configuration/MemorySection.java [hunk 2]
```diff
@@ -787,6 +787,11 @@ public boolean isList(String path) {
         }
 
         List<Object> list = getList(path);
+        
+        if(list == null) {
+            return new ArrayList<Map<String, Object>>(0);
+        }
+
```
Comment (RobinvanderVliet) [inline src/main/java/org/bukkit/configuration/MemorySection.java:794]: And this whitespace. ;)

File: src/main/java/org/bukkit/configuration/MemorySection.java [hunk 3]
```diff
@@ -787,6 +787,11 @@ public boolean isList(String path) {
         }
 
         List<Object> list = getList(path);
+        
+        if(list == null) {
```
Comment (RobinvanderVliet) [inline src/main/java/org/bukkit/configuration/MemorySection.java:791]: And this should be "if (list == null) {".

**Comment by EvilSeph** [comment]
Pulled, thanks!

**Pull request closed without merging**